thingif/src/main/java/com/kii/thingif/query/AggregatedHistoryStatesQuery.java

```diff
@@ -0,0 +1,26 @@
+package com.kii.thingif.query;
+
+import android.support.annotation.NonNull;
+
+public class AggregatedHistoryStatesQuery {
```

Comment:
@yongpingchen 

I think `AggregatedHistoryStateQuery` class is too much.
It may be better to place groupedQury and aggregation as a parameter of `ThingIFAPI#aggregateHistoryStates()` separatedly.